src/latex2sympy/latex2sympy.py

```diff
@@ -673,51 +700,15 @@ def convert_func(self, func):
                 else:  # pragma: no cover
                     raise Exception('Invalid supexpr')
 
+            # handle <trig> methods after parsing `supexpr`
             if name in ['sin', 'cos', 'tan', 'csc', 'sec', 'cot', 'sinh', 'cosh', 'tanh']:
+                # change <trig> -> a<trig> if exponent is -1
                 if func_pow == -1:
                     name = 'a' + name
                     should_pow = False
                 expr = getattr(sympy.functions, name)(arg, evaluate=False)
 
-            if func_pow and should_pow:
-                expr = sympy.Pow(expr, func_pow, evaluate=False)
-
-            return expr
-
-        elif 'func_normal_multi_arg' in func:
```

Comment:
I was able to consolidate some duplicate logic (again, partly pulled from my unknown functions branch) to combine the logic for single arg and multi arg functions